src/utilities/string/checkProtocol.js

```diff
@@ -0,0 +1,12 @@
+import { protocolRegEx, http } from '~/common/constants';
+
+/**
```

Comment:
or http too, right?

Reply:
i am unclear on your comment.can you elaborate?